addons/spreadsheet_account/static/src/index.js

```diff
@@ -21,12 +21,32 @@ cellMenuRegistry.add("move_lines_see_records", {
         const sheetId = position.sheetId;
         const cell = env.model.getters.getCell(position);
         const { args } = getFirstAccountFunction(cell.compiledFormula.tokens);
-        let [codes, date_range, offset, companyId, includeUnposted] = args
-            .map(astToFormula)
-            .map((arg) => env.model.getters.evaluateFormulaResult(sheetId, arg));
-        codes = toString(codes?.value).split(",");
+        let codes, partner_ids = "";
+        let date_range, offset, companyId, includeUnposted = false;
+        const parsed_args = args.map(astToFormula).map(
+            (arg) => env.model.getters.evaluateFormulaResult(sheetId, arg)
+        );
+        if ( cell.content.startsWith("=ODOO.PARTNER.BALANCE") ) {
```

Comment:
this is fishy: any formula that doesn't start with the function you are looking for will not work, even no-op formula like `=1*odoo.partner.balance....`.
You should base everything of the result of `getFirstAccountFunction`: it can return the args as well as the function name.